problems/enumerated/problem_10384/problem_10384.py

```diff
@@ -0,0 +1,172 @@
+from manim import *
+from lib.aramanim import *
+from objects import DScissors
+from .text import *
+
+class Problem10384(Scene):
+	def construct(self):
+		self.wait()
+		self.set_up()
+		self.play(Write(self.task_number))
+		self.wait()
+		self.play(Write(self.condition1))
+		self.wait()
+		self.play(Write(self.condition2))
+		self.wait(1)
+		self.play(
+            VGroup(self.condition1, self.condition2).animate.scale(.8).to_edge(DOWN+LEFT)
+        )
+		self.wait()
+		self.play(Write(self.first_rope_label))
+		self.play(GrowFromEdge(self.first_rope, LEFT))
+		self.play(Write(self.second_rope_label))
+		self.play(GrowFromEdge(self.second_rope, LEFT))
+		self.play(
+			CutIn(self.scissors1),
+			CutIn(self.scissors2)
+		)
+		self.first_rope.become(self.get_part(2.5, color=YELLOW).align_to(self.first_rope, LEFT+UP))
+		self.part2_of_first_rope.next_to(self.first_rope, buff=0)
+		self.add(self.part2_of_first_rope)
+		part2_of_first_rope_label = Tex(part2_of_first_rope_string, font_size=40)
+		part2_of_first_rope_label.add_updater(
+			lambda x: x.next_to(self.part2_of_first_rope, UP)
+		)
+		self.second_rope.become(self.get_part(5).align_to(self.second_rope, LEFT+UP))
+		self.part2_of_second_rope.next_to(self.second_rope, buff=0)
+		self.add(self.part2_of_second_rope)
+		part2_of_second_rope_label = Tex(part2_of_first_rope_string, font_size=40)
+		part2_of_second_rope_label.add_updater(
+			lambda x: x.next_to(self.part2_of_second_rope, UP)
+		)
+		self.play(
+			FadeIn(part2_of_second_rope_label),
+			FadeIn(part2_of_first_rope_label)
+		)
+		self.play(
+			CutOut(self.scissors1),
+			CutOut(self.scissors2)
+		)
+		self.play(
+			self.part2_of_second_rope.animate.shift(RIGHT*3),
+			self.part2_of_first_rope.animate.shift(RIGHT*3)
+		)
+		self.wait()
+		self.play(Transform(self.first_rope, self.get_part(2, color=YELLOW).align_to(self.first_rope, LEFT+UP), run_time=1))
+		self.play(Transform(self.second_rope, self.get_part(6).align_to(self.second_rope, LEFT+UP), run_time=1))
+		self.wait()
+		x3_segments = [*[self.first_rope.copy() for x in range(3)]]
+		self.play(x3_segments[0].animate.align_to(self.second_rope, LEFT+UP))
+		self.play(x3_segments[1].animate.next_to(x3_segments[0], buff=0))
+		self.play(x3_segments[2].animate.next_to(x3_segments[1], buff=0))
+		self.wait()
+		checkmark1 = SVGMobject("objects/SVG_files/check.svg").scale(.3).next_to(self.condition2)
+		self.play(FadeIn(checkmark1))
```

Comment:
Please, import the svg file in `objects/objects.py` python file and then from `objects import Checkmark`